depends/hosts/linux.mk

```diff
@@ -13,12 +13,6 @@ linux_release_CXXFLAGS=$(linux_release_CFLAGS)
 linux_debug_CFLAGS=-O1 -g
 linux_debug_CXXFLAGS=$(linux_debug_CFLAGS)
 
-# https://gcc.gnu.org/onlinedocs/libstdc++/manual/debug_mode.html
-linux_debug_CPPFLAGS=-D_GLIBCXX_DEBUG -D_GLIBCXX_DEBUG_PEDANTIC
```

Comment:
Not sure. This just seems to be removing an option (that otherwise doesn't conflict with the other changes)?

Reply:
I removed this because the same options were enabled by CMake's HARDERNING option when in Debug mode, so this became redundant. However, later I changed HARDENING+Debug to only use `_GLIBCXX_ASSERTIONS` instead of `-D_GLIBCXX_DEBUG -D_GLIBCXX_DEBUG_PEDANTIC` because the latter produced: `undefined reference to mp::Connection::removeSyncCleanup(__gnu_debug::_Safe_iterator...`.

Maybe this should stay. I wonder why using `-D_GLIBCXX_DEBUG -D_GLIBCXX_DEBUG_PEDANTIC` from here does not produce an error...

Reply:
I didn't realize that these flags are used also for the libraries. Dropped these changes.